.ci/travis/travis.sh

```diff
@@ -69,6 +69,7 @@ no-error-test-sbe)
   mvn clean install -Pno-validations
   git clone https://github.com/real-logic/simple-binary-encoding.git
   cd simple-binary-encoding
+  git checkout $(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
I'm sorry, I missed the fact that this is a no error test. I didn't realize this since it isn't inside wercker, where all our other regressions are.
We use master on purpose so we can stay up to date with their latest code.

@romani Do you know what is wrong with their master build?